### ISSUE TITLE
Linux packages need to depend on libusb?

### ISSUE THREAD

**Issue by nickatsegment** [segmentio/aws-okta#109]
Just a hunch, but I suspect this is the case. If so, we should ensure this is documented and captured in builds

**Comment by eculver** [comment]
I was able to confirm this. For Debian, you'll see something like this:

```
# pkg-config --cflags  -- libusb-1.0
Package libusb-1.0 was not found in the pkg-config search path.
Perhaps you should add the directory containing `libusb-1.0.pc'
to the PKG_CONFIG_PATH environment variable
No package 'libusb-1.0' found
pkg-config: exit status 1
```

The fix is easy: `apt-get install libusb-1.0-0-dev` -- I can add some documentation and a Dockerfile to help with this.

**Comment by C8H10N4OO** [comment]
Seems related, but using the packagecloud repo, an `apt-get install aws-okta` does not install `libusb-1.0` automatically.

![image](https://user-images.githubusercontent.com/13878177/58119396-bce70880-7bb7-11e9-93d7-8224fb4eaa63.png)
